src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/DeviceInfo/DeviceInfo.ios.macos.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Foundation;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class DeviceInfo
+	{
+		static string CurrentCultureName => NSLocale.CurrentLocale.LanguageCode;
```

Comment:
This code will be more correct if you have correctly configured the localization in info.plist file

```csharp
NSBundle.MainBundle.PreferredLocalizations?.Length > 0
                ? NSBundle.MainBundle.PreferredLocalizations.First()
                : null;
```

Reply:
Then just ```NSBundle.MainBundle.PreferredLocalizations?.FirstOrDefault();``` should work)

Reply:
in my case, throw is used

Reply:
@dimonovdd I updated implementation so that all preferred languages are returned.

PS: I think your solutions are identical. You check that `Length > 0`, so `First()` will never throw an exception